components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -2010,11 +2035,15 @@ std::string AdsServiceImpl::LoadDataResourceAndDecompressIfNeeded(
 
 void AdsServiceImpl::ShowNotification(
     const std::unique_ptr<ads::AdNotificationInfo> info) {
-  auto notification = CreateAdNotification(*info);
-
-  display_service_->Display(NotificationHandler::Type::BRAVE_ADS,
-      *notification, /*metadata=*/nullptr);
+  LOG(INFO) << "albert AdsServiceImpl::ShowNotification";
+  brave_custom_notification::Notification* notification = (brave_custom_notification::Notification*) CreateAdNotification(*info);
```

Comment:
Please remove cast (you will also need to remove revert raw pointer usage back to smart pointer for CreateAdNotification function)